layouts/partials/site_head.html

```diff
@@ -0,0 +1,337 @@
+    <link rel="stylesheet" href="{{ $hl_theme_dark_css.RelPermalink }}" title="hl-dark" media="print" onload="this.media='all'" {{ if eq ($scr.Get "light") true }}disabled{{end}}>
+  {{ end }}
+
+  {{/* Marketing */}}
+  {{ with site.Params.marketing.verification.google }}
+    <meta name="google-site-verification" content="{{ . }}" />
+  {{- end -}}
+  {{ with site.Params.marketing.verification.baidu }}
+    <meta name="baidu-site-verification" content="{{ . }}" />
+  {{- end -}}
+  {{ with site.Params.marketing.verification.bing }}
+    <meta name="msvalidate.01" content="{{ . }}" />
+  {{- end -}}
+  {{ partial "marketing/google_analytics" . }}
+  {{ partial "marketing/google_tag_manager" . }}
+  {{ partial "marketing/microsoft_clarity" . }}
+  {{ partial "marketing/baidu_tongji" . }}
+
+  {{/* Netlify Identity integration. */}}
+  {{ $use_cms := templates.Exists "wowchemycms/single.wowchemycms_config.yml" | default (site.Params.extensions.cms.enable | default false) }}
+  {{ if .IsHome | and $use_cms }}
+    <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+  {{ end }}
+
+  {{/* RSS Feed */}}
+  {{ with .OutputFormats.Get "RSS" }}
+    <link rel="alternate" href="{{ .RelPermalink }}" type="application/rss+xml" title="{{ site.Title }}" />
+  {{ end }}
+
+  {{/* Progressive Web App (PWA) Manifest */}}
+  {{ with site.Home.OutputFormats.Get "WebAppManifest" }}
+    <link rel="manifest" href="{{ .RelPermalink }}" />
+  {{ end }}
+
+  {{/* Progressive Web App (PWA) Icon */}}
+  <link rel="icon" type="image/png" href="{{ (partial "functions/get_icon" 32).RelPermalink }}" />
+  <link rel="apple-touch-icon" type="image/png" href="{{ (partial "functions/get_icon" 180).RelPermalink }}" />
+
+  <link rel="canonical" href="{{ .Permalink }}" />
+
+  {{/* Get page image for sharing. */}}
+  {{ $sharing_image := resources.GetMatch (path.Join "media" "sharing.*") }}
+  {{ $featured_image := (.Resources.ByType "image").GetMatch "*featured*" }}
+  {{ $avatar_image := (.Resources.ByType "image").GetMatch "avatar*" }}
+  {{ $has_logo := fileExists "assets/media/logo.png" | or (fileExists "assets/media/logo.svg") }}
+  {{ $og_image := "" }}
+  {{ $twitter_card := "summary_large_image" }}
+  {{ if (and (eq .Kind "term") $avatar_image) }}
+    {{/* Match image processing in About widget to prevent generating more images than necessary. */}}
+    {{ $og_image = ($avatar_image.Fill "270x270 Center").Permalink }}
+    {{ $twitter_card = "summary" }}
+  {{ else if $featured_image }}
+    {{ $og_image = $featured_image.Permalink }}
+  
+  {{/* ADDED SECTION - HERE WE LOAD THE PARTIAL TEMPLATE FOR SOCIAL MEDIA CARDS*/}}
+  {{ else if .IsPage }}
+    {{- with partial "ogimage.html" . -}}
+    {{ $og_image = .Permalink }}
+    {{ end -}}
+  {{/* END ADDED SECTION */}}
```

Comment:
This is the only modified section, all the rest is just copy/paste of the original